docs/docs/building-with-components.md

```diff
@@ -122,7 +122,7 @@ document and add external links.
 
 Typically you should omit this from your site as the default html.js file will
 suffice. If you need more control over server rendering, then it's valuable to
-have an html.js.
+have an `html.js`.
 
```

Comment:
Do you know why this same phrase isn't wrapped in backticks earlier in the pragraph?

Reply:
The rule was improperly saying that html.js needed "an". Making it code fixes it (and makes it aligned with other uses elsewhere in the page).